cmd/kogito/command/install/kogito_operator.go

```diff
@@ -94,7 +101,12 @@ func (i *installKogitoOperatorCommand) Exec(cmd *cobra.Command, args []string) e
 		return err
 	}
 
-	install := shared.ServicesInstallationBuilder(i.Client, i.flags.namespace).InstallOperator(true, i.flags.image, i.flags.force)
+	installationChannel := i.flags.channel
```

Comment:
you would here just need to cast it as `KogitoChannelType`

Reply:
incorporated review comments